src/components/settings/EnvironmentVariables/EnvHintButton/index.tsx

```diff
@@ -7,13 +7,15 @@ import { isEnvInitialState } from '@/store/settingsSlice'
 import css from './styles.module.css'
 import AlertIcon from '@/public/images/common/alert.svg'
 import useChainId from '@/hooks/useChainId'
+import useWallet from '@/hooks/wallets/useWallet'
 
 const EnvHintButton = () => {
   const router = useRouter()
   const chainId = useChainId()
+  const wallet = useWallet()
   const isInitialState = useAppSelector((state) => isEnvInitialState(state, chainId))
 
-  if (isInitialState) {
+  if (isInitialState || wallet) {
```

Comment:
Why do we not show any hints if a wallet is connected?
If I now for instance change my tenderly simulation URL the hint will disappear as soon as I connect a wallet.

Reply:
Good point. I've adjusted the logic to preserve the Tenderly warning.

Reply:
After discussion, I've reverted this. We can revisit it later if necessary.